spec/models/tag_group_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe TagGroup, type: :model do
+  let(:project) { build :project, :with_tag}
+  let(:team) { build :team}
+
+  context "A project does have a tag_group" do
+    it { expect(project.tag_group.team_id).to eq(6) }
+    it { expect(project.tag_group.name).to eq('my-tag') }
+    it { expect(project.tag_group.description).to eq('awesome tag') }
```

Comment:
Yep, and it's useful where you don't need to check this file, changing the factory would break this file and the programmer that does that wouldn't know why in a clear way. Prefer to put those values in the test setup as well.